src/test/java/org/folio/rest/impl/CheckinReceivingApiTest.java

```diff
@@ -228,7 +223,7 @@ public void testPostCheckinRevertPhysicalResource() {
       PoLine poLine = pol.mapTo(PoLine.class);
       assertThat(poLine.getCheckinItems(), is(true));
       assertThat(poLine.getReceiptStatus(), is(PoLine.ReceiptStatus.AWAITING_RECEIPT));
-      assertThat(poLine.getReceiptDate(), is(nullValue()));
```

Comment:
This check should remain as is once comment for `updatePoLineReceiptStatus` method from `HelperUtils.java` is addressed